homeassistant/components/konnected/strings.json

```diff
@@ -94,11 +94,15 @@
         "title": "Configure Misc",
         "description": "Please select the desired behavior for your panel",
         "data": {
-          "blink": "Blink panel LED on when sending state change"
+          "blink": "Blink panel LED on when sending state change",
+          "override_api_host": "Override default Home Assistant API host panel URL",
+          "api_host": "Override API host URL (optional)"
```

Comment:
Why have two options. Why not having it be empty be equal to not overriding it?

Reply:
I was struggling to get the config options input field to show a default AND allow returning an empty string.

If the default value was passed into the schema, and then deleted completely, the default was still passed back into the flow step.

I put the additional option in as a workaround but then found it actually seemed more explicit/clear than just showing a user a text box that they would normally leave empty.